openmdao/drivers/tests/test_differential_evolution_driver.py

```diff
@@ -0,0 +1,1052 @@
+""" Unit tests for the DifferentialEvolutionDriver Driver."""
+
+import unittest
+import os
+
+import numpy as np
+
+import openmdao.api as om
+from openmdao.drivers.differential_evolution_driver import DifferentialEvolution
+from openmdao.test_suite.components.branin import Branin, BraninDiscrete
+from openmdao.test_suite.components.paraboloid import Paraboloid
+from openmdao.test_suite.components.paraboloid_distributed import DistParab
+from openmdao.test_suite.components.sellar_feature import SellarMDA
+from openmdao.test_suite.components.three_bar_truss import ThreeBarTruss
+
+from openmdao.utils.assert_utils import assert_near_equal
+from openmdao.utils.mpi import MPI
+
+try:
+    from openmdao.vectors.petsc_vector import PETScVector
+except ImportError:
+    PETScVector = None
+
+extra_prints = False  # enable printing results
+
+class TestDifferentialEvolution(unittest.TestCase):
+    def setUp(self):
+        import openmdao.api as om
```

Comment:
You shouldn't need any imports in the setUp function here or in the other TestCase classes.

Reply:
Imports seem to be required in setup() for the tests that are included in the docs using embed-code. I will delete all the others.

Reply:
You can put them in the actual test.  That way the imports also show up in the docs.